doc/source/tune/user-guide.rst

```diff
@@ -295,49 +295,157 @@ When running a hyperparameter search, Tune can automatically and periodically sa
  * use pre-emptible machines (by automatically restoring from last checkpoint)
  * Pausing trials when using Trial Schedulers such as HyperBand and PBT.
 
-To use Tune's checkpointing features, you must expose a ``checkpoint_dir`` argument in the function signature, and call ``tune.checkpoint_dir``:
+Let's cover how to configure your checkpoints storage location, checkpointing frequency, and how to resume from a previous run.
+
+A simple (cloud) checkpointing example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Cloud storage-backed Tune checkpointing is the recommended best practice for both performance and reliability reasons. rsync-based checkpointing is also incompatible with Ray on Kubernetes. If you'd rather checkpoint locally or use rsync based checkpointing, see here.
+
+Prequisities to use cloud checkpointing in Ray Tune for the example below:
+
+Your ``my_trainable`` is either a:
+
+   1. **Model with an existing Ray integration**
+   
+      * XGBoost (`example <https://github.com/ray-project/xgboost_ray#hyperparameter-tuning>`_)
+      * Pytorch (`example <https://docs.ray.io/en/latest/tune/tutorials/tune-pytorch-lightning.html#putting-it-together>`_)
+      * Pytorch Lightning (`example <https://github.com/ray-project/ray_lightning#hyperparameter-tuning-with-ray-tune>`_)
+      * Keras (`example <https://docs.ray.io/en/latest/tune/examples/tune_mnist_keras.html>`_)
+      * Tensorflow (`example <https://docs.ray.io/en/latest/raysgd/raysgd_tensorflow.html#tftrainer-example>`_)
+      * LightGBM (`example <https://docs.ray.io/en/latest/lightgbm-ray.html#hyperparameter-tuning>`_)
+  
+   2. **Custom training function**
+   
+      * All this means is that your function has to expose a ``checkpoint_dir`` argument in the function signature, and call ``tune.checkpoint_dir``. See this example, it's quite simple to do.
```

Comment:
I want to link here to `custom_func_checkpointing.rst`, but not sure how to do that